README.md

```diff
@@ -1,2 +1,119 @@
 # Red Hat Icons
 Official Red Hat Brand Icons
+
+
+## Licensing
+
```

Comment:
Maybe we should check of the JS modules should stay MIT, but retain the license comment in the exported svg  node

Reply:
Reworded the README a bit for this section, let me know if you feel that better applies.